src/app/submission/shared/submission.model.ts

```diff
@@ -614,8 +618,22 @@ export class Sections extends HasUpdates<UpdateEvent> {
         return s;
     }
 
-    remove(): void {
-        // TODO
+    remove(section: Section): void {
```

Comment:
There is still a possibility to call section.sections.remove(...) from anywere in the code.